content/docs/pipelines/sdk/build-component.md

```diff
@@ -62,55 +62,66 @@ specify how to map the content of local files to component outputs.
 
 ## Create a Python class for your component
```

Comment:
Update title? `Create a Python function to wrap your component` perhaps?

Reply:
That reads well to me, updated.